### DISCUSSION TITLE
Welcome to garo-charger-glb Discussions!

### DISCUSSION THREAD

**Discussion by fgeorgsson** [flexergi/homey-garo-charger-glb#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PACE2PACE** [comment]
Hi Fredric. You recently asked for input for you GARO app for homey and requested support with rest response. I am not very experienced with rest so I hope this was what you requested.
I am not sure if this will help you add the actual current use for the load balansing unit and if not I will support if I can.

Command: http://xxx.xxx.xxx.xxx:8080/servlet/rest/chargebox/status
Response {"serialNumber":xxxxxxxxxx,"ocppState":null,"connectedToInternet":true,"freeCharging":false,"ocppConnectionState":null,"connector":"NOT_CONNECTED","mode":"ALWAYS_OFF","currentLimit":16,"factoryCurrentLimit":32,"switchCurrentLimit":16,"powerMode":"OFF","currentChargingCurrent":0,"currentChargingPower":0,"accSessionEnergy":0,"sessionStartTime":null,"chargeboxTime":"14:16","accSessionMillis":0,"latestReading":4048000,"chargeStatus":0,"updateStatus":{"serialsToUpdate":[],"serialsUpdated":[],"currentlyUpdating":-1,"currentProgress":-1,"failedUpdate":"NO_ERROR"},"currentTemperature":29,"sessionStartValue":-1,"nrOfPhases":1,"slaveControlWarning":false,"supportConnectionEnabled":false,"datetimeConfigured":true,"pilotLevel":6,"mainCharger":{"reference":null,"serialNumber":xxxxxxxxx,"lastContact":1666441005119,"online":false,"loadBalanced":false,"phase":0,"productId":131,"meterStatus":0,"meterSerial":"125388V","chargeStatus":0,"pilotLevel":6,"accEnergy":4048000,"firmwareVersion":8,"firmwareRevision":3,"wifiCardStatus":2,"connector":"NOT_CONNECTED","accSessionEnergy":0,"sessionStartValue":-1,"accSessionMillis":0,"sessionStartTime":null,"currentChargingCurrent":0,"currentChargingPower":0,"nrOfPhases":1,"twinSerial":-1,"cableLockMode":0,"minCurrentLimit":6,"dipSwitchSettings":8090,"cpuType":2004993,"updateable":false},"twinCharger":null}.

If I understand correctly this is how to access the external loadbalanser values.
Request: 
xxx.xxx.xxx.xxx:8080/servlet/rest/chargebox/meterinfo/EXTERNAL

  Answer:
{"success":0,"accEnergy":19307400,"phase1Current":10,"phase2Current":17,"phase3Current":14,"phase1InstPower":0,"phase2InstPower":0,"phase3InstPower":0,"readTime":1666442893639,"gridNetType":"UNKNOWN","meterSerial":"079095V","type":341,"apparentPower":0}

Be aware that the current values are a factor of 10 too big to not need to use decimals I presume so 17 meens 1.7 A.

/Martin
